sky/provision/docker_utils.py

```diff
@@ -218,14 +218,21 @@ def initialize(self) -> str:
                     f'{specific_image}')
         container_running = self._check_container_status()
         if container_running:
-            running_image = (self._run(
-                check_docker_image(self.container_name, self.docker_cmd)))
+            running_image = self._run(
+                check_docker_image(self.container_name, self.docker_cmd))
             if running_image != specific_image:
                 logger.error(
                     f'A container with name {self.container_name} is running '
                     f'image {running_image} instead of {specific_image} (which '
                     'was provided in the YAML)')
         else:
+            # Edit docker config first to avoid the know issue in
```

Comment:
Would be nice to elaborate the issue in the comment. Something like 
```suggestion
            # Edit docker config first to avoid disconnecting the container from GPUs when a systemctl command is called (a known issue with docker).
```

Reply:
Good point! Added.